module.py

```diff
@@ -16,6 +16,7 @@ def initialize(self, context):
         self.proxy_authorizer_class(Authorizer)
 
         Authenticator._configure_users(context.users)
+        Authenticator._configure_unhashed(context.allow_unhashed_passwords)
```

Comment:
Typically with these I think we just have one `configure` method that takes these parameters or the context I think. Not a huge deal but it might set up for easier work in the future.

```python
class Authenticator:
    def _configure(cls, users, allow_unhashed):
        ...

Authenticator._configure(users=context.users, allow_unhashed=context.allow_unhashed_passwords)
```

Reply:
I was originally going to try this but decided against it for simplicity. But I think it makes sense since that configure will only affect the Authenticator class. I'll go with your suggestion!